infrastructure/k8s/helm-chart/eoapi/templates/raster/manifest.yaml

```diff
@@ -44,29 +45,74 @@ spec:
             cpu: "256m"
             memory: "3072M"
         env:
-          # NOTE: there's no reason we need to use a `ConfigMap` here to get os env vars into the pod.
+          # NOTE: there's no reason we need to use a `ConfigMap` or `Secret` here to get os env vars into the pod.
           # we could just template them out here immediately with `value: $_` but this allows us
           # to store them in k8s intermediately and change them and then bounce deploys if needed
+          {{- range $envKey, $_ := .Values.db.settings.secrets }}
```

Comment:
If you are using ALL the secrets here for the env, is it possible to use `envFrom` and reference the whole secret, like we do here? https://github.com/IFRCGo/go-api/blob/terraform-prod/deploy/helm/ifrcgo-helm/templates/api/deployment.yaml#L46

No strong preference though.

Reply:
oh yeah, if forgot about this option. I'll get around to this change for sure